.github/workflows/test-python.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Get full Python version
       id: full-python-version
-      run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
+      run: echo "version={$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")}" >> $GITHUB_OUTPUT
```

Comment:
The `{` and `}` seem out of place. I think you may be taking [the guidance](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) to literally.

Reply:
Fixed.